cloudsql-mysql-plugin/widgets/CloudsqlMysql-batchsource.json

```diff
@@ -0,0 +1,162 @@
+      "label": "Credentials",
+      "properties": [
+        {
+          "widget-type": "textbox",
+          "label": "Username",
+          "name": "user",
+          "widget-attributes": {
+            "placeholder": "The username to use to connect to the CloudSQL database"
+          }
+        },
+        {
+          "widget-type": "password",
+          "label": "Password",
+          "name": "password",
+          "widget-attributes": {
+            "placeholder": "The password to use to connect to the CloudSQL database"
+          }
+        }
+      ]
+    },
+    {
+      "label": "CloudSQL Properties",
+      "properties": [
+        {
+          "widget-type": "radio-group",
+          "label": "CloudSQL instance Type",
+          "name": "instanceType",
+          "widget-attributes": {
+            "layout": "inline",
+            "default": "public",
+            "options": [
+              {
+                "id": "public",
+                "label": "Public"
+              },
+              {
+                "id": "private",
+                "label": "Private"
+              }
+            ]
+          }
+        },
+        {
+          "widget-type": "textbox",
+          "label": "Connection Name",
+          "name": "connectionName",
+          "widget-attributes": {
+            "placeholder": "CloudSQL instance connection name"
+          }
+        },
+        {
+          "widget-type": "textarea",
+          "label": "Import Query",
+          "name": "importQuery",
+          "widget-attributes": {
+            "rows": "4"
+          }
+        },
+        {
+          "widget-type": "get-schema",
+          "widget-category": "plugin"
+        }
+      ]
+    },
+    {
+      "label": "Advanced",
+      "properties": [
+        {
+          "widget-type": "textarea",
+          "label": "Bounding Query",
+          "name": "boundingQuery",
+          "widget-attributes": {
+            "rows": "4"
+          }
+        },
+        {
+          "widget-type": "textbox",
+          "label": "Split Column",
+          "name": "splitBy"
+        },
+        {
+          "widget-type": "textbox",
+          "label": "Number of Splits",
+          "name": "numSplits",
+          "widget-attributes": {
+            "default": "1"
+          }
+        },
+        {
+          "widget-type": "keyvalue",
+          "label": "Connection Arguments",
+          "name": "connectionArguments",
+          "widget-attributes": {
+            "showDelimiter": "false",
+            "key-placeholder": "Key",
+            "value-placeholder": "Value",
+            "kv-delimiter": "=",
+            "delimiter": ";"
+          }
+        }
+      ]
+    }
+  ],
+  "outputs": [
+    {
+      "name": "schema",
+      "widget-type": "schema",
+      "widget-attributes": {
+        "schema-types": [
+          "boolean",
+          "int",
+          "long",
+          "float",
+          "double",
+          "bytes",
+          "string"
```

Comment:
output schema should support more types right? How about array, record, date, time, timestamp?

Reply:
Not entirely sure about this. I don't see those data types in the output schema for the existing MySQL plugin.

The plugin still seems to support those data types even though it's not mentioned in the json.